metrics/internal/collectors/registry.go

```diff
@@ -4,13 +4,14 @@ import (
 	"time"
 
 	"github.com/prometheus/client_golang/prometheus"
-	internalcache "github.com/red-hat-storage/ocs-operator/metrics/internal/cache"
-	"github.com/red-hat-storage/ocs-operator/metrics/internal/options"
 	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 	rookclient "github.com/rook/rook/pkg/client/clientset/versioned"
 	corev1 "k8s.io/api/core/v1"
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/cache"
+
+	internalcache "github.com/red-hat-storage/ocs-operator/metrics/internal/cache"
+	"github.com/red-hat-storage/ocs-operator/metrics/internal/options"
```

Comment:
This isn't required.

Reply:
This is based on the `gofmt` import arrangement (PTAL below). Should I revert it?

<details>
<summary>Settings</summary>

![Screenshot from 2022-10-18 11-02-26](https://user-images.githubusercontent.com/33557095/196343894-1df00367-49a6-40e3-9130-0016e0919e08.png)


</details>